### PR TITLE
feat(relativePathBaseUrl): Add support for prepending a base URL

### LINKED ISSUE #536: provide "topurl" to convert relative URL to the full one
May be it is already implemented but I could not determine by looking at the source, and please pardon my naivete in JavaScript

My example is http://datasets.datalad.org/?dir=/labs/haxby/attention where actual url for the page remains http://datasets.datalad.org/ while I am "browsing" nested directories.  README.md is getting rendered by showdown and includes relative URLs such as `stimuli/README.md`.  I would like to inform `Converter` that top URL for those should be http://datasets.datalad.org/labs/haxby/attention .  Is that may be already possible?

Thank you in advance for the clarification!

### PR DESCRIPTION
**Pull request by childish-sambino**
This feature enables support for prepending a base URL to relative paths in
links and images when converting Markdown to HTML.

Closes #536

### CODE REVIEW

**Comment by tivie** [comment]
Thanks for your contribution. I will have a look as soon as possible

**Comment by childish-sambino** [comment]
@tivie Could you take a look when you have a chance?

File: src/options.js [hunk 1]
```diff
@@ -160,7 +160,12 @@ function getDefaultOpts (simple) {
       defaultValue: false,
       description: 'Split adjacent blockquote blocks',
       type: 'boolean'
-    }
+    },
+    relativePathBaseUrl: {
+      defaultValue: false,
+      describe: 'Prepends a base URL to relative paths',
```
[the remark this answers is not in the document]
Comment (childish-sambino) [reply src/options.js]: The file is mixed with both. It's unclear which is proper.

**Comment by twk-b** [comment]
this would be super useful. what needs to happen to get it over the finish line?

**Comment by Ghustavh97** [comment]
Any new updates on this PR?

**Comment by SyntaxRules** [comment]
Thanks for the PR @childish-sambino !

**Pull request merged**